### PR TITLE
[ i18n] Cleanup

### PR DESCRIPTION
**Pull request by delphiactual**
General cleanup of the i18n file.
All outer quotes are now double quotes.
Moved trailing curly braces to the next line.
Untranslated a filter keyword.
Translate the automatic loadout toasters

Added new keys
 - `BungieManifest.DevVersion`
 - `BungieManifest.NoCookies`
 - ~~`Help.UpgradeChrome`~~ language is not set when this is checked
 - `Infusion`

### CODE REVIEW

**Comment by bhollis** [comment]
In general it's nice to put cleanups/reformatting in a separate PR from real changes.

File: app/scripts/move-popup/dimMovePopup.directive.js [hunk 1]
```diff
@@ -21,7 +21,7 @@
         '  <dim-move-amount ng-if="vm.item.amount > 1 && !vm.item.notransfer" amount="vm.moveAmount" maximum="vm.maximum"></dim-move-amount>',
         '  <div class="interaction">',
         '    <div class="locations" ng-repeat="store in vm.stores | sortStores:vm.settings.characterOrder track by store.id">',
-        '      <div class="move-button move-vault" alt="{{::store.name}}" title="{{::store.name}}" ',
+        '      <div class="move-button move-vault" translate-attr="{ title: \'Bucket.Vault\', alt: \'Bucket.Vault\' }" ',
```
Comment (bhollis) [inline app/scripts/move-popup/dimMovePopup.directive.js]: `store.name` should already be localized for the vault, right? The other stores are, we should be consistent about that.

Comment (delphiactual) [inline app/scripts/move-popup/dimMovePopup.directive.js]: It does not translate it on hover without this change, not sure why.

Comment (bhollis) [inline app/scripts/move-popup/dimMovePopup.directive.js]: Looks like it's because it used bind-once? Not sure. `store.name` should be translated.

Comment (delphiactual) [inline app/scripts/move-popup/dimMovePopup.directive.js]: it's not tho, I removed the bind once, set language to italian and still shows vault as title on hover

**Comment by delphiactual** [comment]
@Korben85, @crulplay, @simonefranza, @JaviG1105, @brunnopleffken, @yannick:
Can you guys please review the change to Compare.All and Compare.Archetype and make sure they are correct?

**Comment by delphiactual** [comment]
@kyleshay By changing the wording of `Compare all {{type}}` to `{{type}} comparisons` we remove the need to pluralize {{type}}.

**Comment by kyleshay** [comment]
Is there anything else left for this?

**Comment by delphiactual** [comment]
AFAIK this is complete unless we want to track down why store.name is not translated for vault...

**Comment by bhollis** [comment]
I do want to track that down.

**Comment by delphiactual** [comment]
@bhollis, in `dimStoreService.factory.js` on this [line](https://github.com/DestinyItemManager/DIM/blob/dev/app/scripts/services/dimStoreService.factory.js#L359) and this [line](https://github.com/DestinyItemManager/DIM/blob/dev/app/scripts/services/dimStoreService.factory.js#L362). Translation language has not been set yet, so i18n is defaulting to `en`.

**Comment by delphiactual** [comment]
@bhollis the keyword was never updated after vault got moved under buckets in the i18n file. This is now fixed. Ready for merge.

**Pull request merged**